src/vapid-helper.js

```diff
@@ -90,6 +96,48 @@ function validatePrivateKey(privateKey) {
 }
 
 /**
+ * Calculates the total expiration by adding
+ * value of Date.now() with the seconds passed
+ *
+ * @param {Number} numSeconds Number of seconds to calculate expiration
+ * @return {Number} expiration in seconds
+ */
+function calculateExpiration(numSeconds) {
+  return Math.floor(Date.now() / 1000) + numSeconds;
+}
+
+/**
+ * Validates the Expiration based on the VAPID Spec
+ *
+ * `expiration` is not validated with a `minimum` value,
+ * since the time at validation is going to be different
+ * than at the time of creation of the custom expiration value
+ */
+function validateExpiration(expiration) {
```

Comment:
@gauntface Sorry for this confusion, I have removed the assumption that it could be string at the first place, as It was confusing, and the Comment Signature for the getVapidHeaders also suggested that expiration must be `integer` so is it still needed?  Since it's no longer being parsed from string, we might not need two terms. However, I am still open to add `expirationFromSeconds` if that makes sense. Would love your opinion on it. Thanks :+1: